README.md

```diff
@@ -48,9 +48,9 @@ npm install
 Troubleshooting
 ---
 If you have any problems installing the ui-toolkit then please view the
-[troubleshooting guide](https://github.com/holidayextras/ui-toolkit/blob/master/TROUBLESHOOTING.md) which contains common errors and
+[troubleshooting guide](/blob/master/TROUBLESHOOTING.md) which contains common errors and
```

Comment:
I think from one markdown file to another you can just call the file `TROUBLESHOOTING` and it will go there